cardano-testnet/src/Testnet/EpochStateProcessing.hs

```diff
@@ -60,29 +59,29 @@ waitForGovActionVotes
   -> EpochInterval -- ^ The maximum wait time in epochs.
   -> m ()
 waitForGovActionVotes epochStateView maxWait = withFrozenCallStack $ do
-  mResult <- watchEpochStateUpdate epochStateView maxWait checkForVotes
-  when (isNothing mResult) $
+  (cond, ()) <- watchEpochStateUpdate epochStateView maxWait checkForVotes
+  when (cond == ConditionNotMet) $
```

Comment:
I'm not a big fan of those `==` on a union type, for long term maintenance; but in this specific case I think it's fine. It's unlikely we'll add a third case to `LedgerStateCondition`.